lib/browser/guest-view-manager.js

```diff
@@ -115,30 +115,30 @@ const createGuest = function (embedder, params) {
   // Dispatch events to embedder.
   const fn = function (event) {
     guest.on(event, function (_, ...args) {
-      sendToEmbedder('ELECTRON_GUEST_VIEW_INTERNAL_DISPATCH_EVENT', event, ...args);
+      sendToEmbedder('GUEST_VIEW_INTERNAL_DISPATCH_EVENT', event, ...args);
```

Comment:
can these be enums too?

Reply:
done in TypeScript, I don't think we can use `const enum` in JavaScript

Reply:
they cannot be used here in JavaScript code, needs to be converted to TypeScript

Reply:
or I can change the `enum` to be non-`const`